app/controllers/api/v2/reserves_controller.rb

```diff
@@ -39,7 +39,7 @@ def reserve
         respond_to do |format|
           format.json { render :json => @hosts }
           format.yaml { render :text => @hosts.to_yaml }
-          format.html { not_found }
+          format.html { render :json => hosts }
```

Comment:
Hi David,

unfortunately I don't have any clue about Ruby prog. language. I don't know what semantics change means `@hosts` instead of `hosts`. My guess comes from other outcomes (other methods implemented there) from reserves_controller.rb.

I can see there that there is a mixture of either `hosts` or `@hosts`, from quick search I understand that `@hosts` is an instance variable under `self`, but in that case I don't understand if method `reserve` is correctly implemented too.

Do you know what is correct in this case?

Thank you, Pavel